foundations/javascript_basics/DOM_manipulation_and_events.md

```diff
@@ -323,7 +323,7 @@ btn.addEventListener('click', function (e) {
 ~~~
 \*_Note that `function (e)` is a callback from addEventListener. Further explanation of callbacks can be found [HERE](https://dev.to/i3uckwheat/understanding-callbacks-2o9e)._
 
-The `e` in that function is an object that references the __event__ itself.  Within that object you have access to many useful properties and functions such as which mouse button or key was pressed, or information about the event's __target__ - the DOM node that was clicked.
+The `e` in that function is an object that references the __event__ itself.  Within that object you have access to many useful properties and methods such as which mouse button or key was pressed, or information about the event's __target__ - the DOM node that was clicked.
```

Comment:
Something like this would be passable IMO.

```suggestion
The `e` in that function is an object that references the __event__ itself.  Within that object you have access to many useful properties and methods (functions that live inside an object) such as which mouse button or key was pressed, or information about the event's __target__ - the DOM node that was clicked.
```